Set/Set/Controller/Card.swift

```diff
@@ -12,6 +12,7 @@ struct Card {
     let shade: Shade
     let shape: Shape
     let number: Number
+    let id: Int
```

Comment:
In future I would recommend you to use `Identifiable` protocol and access object ID  type by `Card.ID` to abstract its implementation